config/main.py

```diff
@@ -315,6 +316,7 @@ def _restart_services():
         'lldp',
         'snmp',
         'dhcp_relay',
+        'hostcfgd',
     ]
```

Comment:
You might have to put hostcfgd at the beginning of the list?

Reply:
hostcfgd has no dependency relationship with any other daemon. Its only job is to make TACPLUS auth related updates (/etc/pam.d/*,  /etc/nsswitch.conf & "/etc/tacplus_nss.conf"). hostcfgd's only requirement is accessibility to redis.

So the order does not matter.